operator/metrics/metrics.go

```diff
@@ -127,6 +154,34 @@ func registerMetrics() []prometheus.Collector {
 	}, []string{LabelOutcome})
 	collectors = append(collectors, IdentityGCRuns)
 
+	CiliumEndpointSliceDensity = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "number_of_ceps_per_ces",
+		Help:      "The number of CEPs batched in a CES",
+	})
+	collectors = append(collectors, CiliumEndpointSliceDensity)
+
+	CiliumEndpointsChangeCount = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "number_of_cep_changes_per_ces",
```

Comment:
Have you run promtool against these metrics?

Typically rather than `number_of_foo` prefix prometheus encourages `foo_count` suffix. It's a nit, but I'd rather try to keep metrics consistent with prometheus best practices if we can.

Feel free to ignore if promtool is OK with these metrics. The ones below with seconds/total suffix below look like they follow the standard naming convention.

Reply:
I didn't use promtool against these metrics, i had used grafana to check metrics. If its quicker i'll run promtool, or follow up an in another PR, unless if it's high priority.

Reply:
promtool is basically just a linter that you can run against a live prometheus API to give you feedback on whether the metrics are following best practices. Shouldn't take too long, but would be nice to get these names nailed down by the final release.

Reply:
SG, thanks. Let you know results with promtool.

Reply:
Interesting found couple of issues.,

```
gobinathk@cs-master-1:~$ curl -s localhost:6942/metrics | promtool check metrics
cilium_operator_identity_gc_entries_total non-counter metrics should not have "_total" suffix
cilium_operator_identity_gc_runs_total non-counter metrics should not have "_total" suffix
```

Reply:
@joestringer Above reported metrics aren't part of CES related changes, WDYT fix it in another PR ?

Reply:
yeah SGTM. We should really just add this to the linters in the github actions so it's easier to trigger this, but it's just a bit lower on the TODO list.

Reply:
Here is [PR](https://github.com/cilium/cilium/pull/17789) fixes promtool lint issues